submit.py

```diff
@@ -68,11 +69,13 @@ def write_general_header(self, f, mem=3000, walltime_hours=14,
             outdir = os.path.join(os.getcwd(),'out')
             if not os.path.isdir(outdir):
                 os.mkdir(outdir)
-            self.write_line(f, "#PBS -o %s/${PBS_JOBID}.out"%outdir)
-            self.write_line(f, "#PBS -e %s/${PBS_JOBID}.err"%outdir)
+            self.write_line(f, "#PBS -o %s/${PBS_JOBID}.out" % outdir)
+            self.write_line(f, "#PBS -e %s/${PBS_JOBID}.err" % outdir)
```

Comment:
`${PBS_JOBID}` has the array index in it. It looks something like `1387076[4].parallel-admin.out-4`